packages/contracts/tests/js/StabilityPoolTest.js

```diff
@@ -1296,70 +1298,6 @@ contract('StabilityPool', async accounts => {
       assert.isTrue(C_ZEROBalance_After.eq(C_ZEROBalance_Before));
     });
 
-    it("provideToSP(), topup: depositor receives ZERO rewards", async () => {
```

Comment:
why was this test removed instead of updated?

Reply:
Accidentally removed.
Brought back and updated